api.go

```diff
@@ -66,14 +66,27 @@ func NewPathApi(ipfspath string) (*HttpApi, error) {
 	return NewApi(a)
 }
 
+// NewAddrApi constructs new HttpApi by pulling api address.
+// Api address should be set like /ip4/127.0.0.1/tcp/5001.
+func NewAddrApi(apiAddr string) (*HttpApi, error) {
```

Comment:
I'm not convinced this function is worth having. Parsing the multiaddr then calling `NewApi` isn't difficult.

Reply:
Yes,use isn't difficult.
But it took me a little time to investigate how to use this. 
So I think it can be a bit simpler.
Maybe there will be a better way later.